src/platform/site-wide/user-nav/sass/user-nav.scss

```diff
@@ -323,7 +323,9 @@ span.sidelines {
 #create-account a {
   border-top: 1px solid $color-gray-light;
   border-bottom: 1px solid $color-gray-light;
-  padding-bottom: 1.16em;
+  padding: 1.16em 0;
+  line-height: 100%;
+  width: 100%;
```

Comment:
We have a design system class we should use here instead: `vads-u-width--full` cc @k80bowman  

https://design.va.gov/utilities/height-and-width